.vscode/launch.json

```diff
@@ -34,7 +43,13 @@
       }
     },
     {
-      "name": "Debug Lsp Server",
+      // Attaching LLDB by Pid to lsp server
+      // NOTE: it requires root permission, also for better debugging
+      // if you want more than just assembler representation for ex: 
+      // walk in file, you need debugging symbols.
+      // Project can build with it, for set - chane inside
+      // ./Cargo.toml field debug 
+      "name": "Attach To Server",
```

Comment:
```suggestion
      // Used to attach LLDB to a running LSP server.
      // NOTE: Might require root permissions (see `kernel.yama.ptrace_scope`).
      // Don't forget to enable debug info in `Cargo.toml` beforehand. 
```

Reply:
This didn't come out right, sorry.

Reply:
This has a more precise name. When you debug, do you usually expect a launch with debugger